src/background/index.js

```diff
@@ -0,0 +1,11 @@
+/* globals chrome */
+
+chrome.commands.onCommand.addListener((command) => {
```

Comment:
Background script listens for keybindings and uses message passing to notify the client of the action

Reply:
this works on both Firefox and chrome. should we consider using `webextension-polyfill` instead for consistency?

Reply:
Awesome. Let's do this in a future PR (I need to test in Firefox still anyway!)